internal/dsync/drwmutex.go

```diff
@@ -247,6 +253,26 @@ func (dm *DRWMutex) startContinousLockRefresh(lockLossCallback func(), id, sourc
 	}()
 }
 
+func forceUnlock(ctx context.Context, ds *Dsync, id string) {
+	restClnts, _ := ds.GetLockers()
+
+	var wg sync.WaitGroup
+	for index, c := range restClnts {
+		wg.Add(1)
+		// Send refresh request to all nodes
+		go func(index int, c NetLocker) {
+			defer wg.Done()
+			args := LockArgs{
+				UID: id,
+			}
+			ctx, cancel := context.WithTimeout(ctx, drwMutexForceUnlockCallTimeout)
+			defer cancel()
```

Comment:
can you use a single context here there is no need for multiple contexts right? @vadmeste

Reply:
> can you use a single context here there is no need for multiple contexts right? @vadmeste

I would prefer a different 30 seconds timeout, no harm in that @harshavardhana I believe. This is not a hotpath. This will only happen when a lock is not refreshed.

Reply:
okay @vadmeste - how do I test this?

Reply:
> okay @vadmeste - how do I test this?

To test we need to insert fake code:

```diff
diff --git a/cmd/lock-rest-server.go b/cmd/lock-rest-server.go
index bebd00ca4..f53480ada 100644
--- a/cmd/lock-rest-server.go
+++ b/cmd/lock-rest-server.go
@@ -103,6 +103,11 @@ func (l *lockRESTServer) RefreshHandler(w http.ResponseWriter, r *http.Request)
                return
        }
 
+       if time.Since(globalBootTime) > 1*time.Minute && time.Since(globalBootTime) < 3*time.Minute {
+               time.Sleep(time.Hour)
+               return
+       }
+
        args, err := getLockArgs(r)
        if err != nil {
                l.writeErrorResponse(w, err)
```

Without this PR, after two minutes, you can start to see a stale lock. But it is fine with this lock.